src/components/proposal/ProposalTableOfficer.tsx

```diff
@@ -277,30 +273,17 @@ const ProposalTableOfficer: React.FC<ProposalTableOfficerProps> = ({
     const iconButtonStyle = { padding: '7px' };
 
     return (
-      <>
-        <Tooltip title="View proposal">
-          <IconButton
-            data-cy="view-proposal"
-            onClick={() => {
-              setUrlQueryParams({ reviewModal: rowData.primaryKey });
-            }}
-            style={iconButtonStyle}
-          >
-            <Visibility />
-          </IconButton>
-        </Tooltip>
-        <Tooltip title="Download proposal as pdf">
-          <IconButton
-            data-cy="download-proposal"
-            onClick={() =>
-              downloadPDFProposal([rowData.primaryKey], rowData.title)
-            }
-            style={iconButtonStyle}
-          >
-            <GetAppIcon />
-          </IconButton>
-        </Tooltip>
-      </>
+      <Tooltip title="View proposal">
+        <IconButton
+          data-cy="view-proposal"
+          onClick={() => {
+            setUrlQueryParams({ reviewModal: rowData.primaryKey });
+          }}
+          style={iconButtonStyle}
```

Comment:
IconButton has `sx` property, which you could use. Not sure what is the best practice here.

Reply:
Yes probably this is left from the previous code. I can change it I think sx would work better without defining another variable.